README.md

```diff
@@ -309,6 +309,12 @@ We currently are using Open Telemetry for distributed tracing, with our OTEL exp
 
 You should now be at our New Relic dashboard where all our OTEL metrics are being displayed.
 
+## Launch Darkly
+
+We currently use the CMS Federal (.us) install of Launch Darkly to manage our feature flags. This can be accessed through the [CMS SSO portal](https://mo-idp.cms.gov) if you have the appropriate EUA job code.
```

Comment:
Is there a real url we can put here that redirects through the mo-idp url? Or do you have to go to the portal to get in (I still haven’t gotten access yet)

Reply:
This is what they gave to us, but it looks like if I hit app.launchdarkly.us and give it our `@teamtrussworks` email address it'll redirect for us. I'll update these docs.